Microsoft/WebTemplateStudio
# Added unique class names for svgs

Summary of changes:
- Changed all svg names to unique names by appending the file name to the class name.

How to Test:
Build, then run extension. Walk through the wizard and ensure that SVGs still display in both light and dark modes.

Screenshot:
Before
![Planet turned yellow and grey due to azure functions](https://user-images.githubusercontent.com/21695878/62090108-85808200-b220-11e9-8755-cd7e7289f6b5.png)
After
![Globe keeping original colours even with the addition of the azure functions svg](https://user-images.githubusercontent.com/32103099/62553162-dd307600-b83c-11e9-9108-9790697a2633.png)


closes #967
